spec/features/officing/residence_spec.rb

```diff
@@ -96,4 +100,7 @@
 
   end
 
+  after do
```

Comment:
Can you move this block right after the `background` block? It's easier to understand what's going to happen that way :smile_cat:

Amend if you can, there's no need to create a new commit for this.

Reply:
Done!